src/db/models/user.js

```diff
@@ -0,0 +1,49 @@
+import { hashPassword } from '../../helpers/utils';
```

Comment:
What is the difference between an `Author` and a `User`?
Any reason why they have different models and tables?

Reply:
the concept is for admin users to list books on the platform that is why `user` is different from `author`